doc/developers/contributing.rst

```diff
@@ -597,8 +597,8 @@ to ``zero_one`` and call ``zero_one_loss`` from that function::
         # actual implementation
         pass
 
-    @deprecated("Function 'zero_one' has been renamed to "
-                "'zero_one_loss' and will be removed in release 0.15."
+    @deprecated("Function 'zero_one' was renamed to 'zero_one_loss' "
+                "in version 0.13 and will be removed in release 0.15. "
```

Comment:
You mean the space after '0.15'? If I remove that won't the sentences be run together?

Here's a shorted example. With trailing space:

    In [1]: ("'zero_one_loss' "
             "0.15. "
             "Default behavior is changed")
    Out[1]: "'zero_one_loss' 0.15. Default behavior is changed"
    
Without:

    In [2]: ("'zero_one_loss' "
              "0.15."
              "Default behavior is changed")
    Out[2]: "'zero_one_loss' 0.15.Default behavior is changed"